cmd/virt-launcher/virt-launcher.go

```diff
@@ -444,6 +444,16 @@ func main() {
 		}
 	}
 
+	terminateIstioProxyIfPresent := func() {
+		if istioProxyPresent() {
+			err := exec.Command("curl", "-XPOST", "http://127.0.0.1:15020/quitquitquit").Run()
+			if err != nil {
+				log.Log.Error("Failed to request Istio proxy termination")
+			}
+		}
```

Comment:
if this is just an http request, make an http request directly instead executing curl.

Reply:
Done